src/Core/Services/Implementations/HandlebarsMailService.cs

```diff
@@ -260,7 +260,7 @@ public async Task SendTrialInitiationEmailAsync(string userEmail)
             SiteName = _globalSettings.SiteName
         };
         await AddMessageContentAsync(message, "TrialInitiation", model);
-        message.Category = "Welcome";
+        message.Category = "TrialInitiation";
```

Comment:
❓ Sorry I think I might be missing something; can you run me through how this addresses the double emailing bug?

Reply:
revert it to welcome. I was still pushing changes then. Please review again